src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,86 @@
+|          f32           | The base ABI uses 32-bit FPRs for parameter passing. (lp64f ilp32f)|
+|          sf            | The base ABI uses no FPR for parameter passing. (lp64s ilp32s)     |
+
+|`ABI type(Base ABI/ABI extension)`| `C library` | `kernel` |          `target tuple`          |
+|----------------------------------|-------------|----------|----------------------------------|
+|           lp64d/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf64 |
+|           lp64f/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf32 |
+|           lp64s/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnusf  |
+|           ilp32d/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf64 |
+|           ilp32f/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf32 |
+|           ilp32s/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnusf  |
+|           lp64d/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf64|
+|           lp64f/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf32|
+|           lp64s/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslsf |
+|           ilp32d/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf64|
+|           ilp32f/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf32|
+|           ilp32s/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslsf |
+
+## Target maintainers
+
+- zhaixiaojuan, `zhaixiaojuan@loongson.cn`, https://github.com/zhaixiaojuan
+
+## Requirements
+
+This target is cross-compiled.
+A GNU toolchain for LoongArch target is required.  It can be downloaded from https://github.com/loongson/build-tools/releases, or built from the source code of GCC (12.1.0 or later) and Binutils (2.39 or later).
+
+## Building the target
+
+The target can be built by enabling it for a `rustc` build.
+
+```toml
+[build]
+target = ["loongarch64-unknown-linux-gnuf64"]
+```
+
+Make sure `loongarch64-unknown-linux-gnu-gcc` can be searched from the directories specified in`$PATH`. Alternatively, you can use GNU LoongArch Toolchain by adding the following to `config.toml`:
+
+```toml
+[target.loongarch64-unknown-linux-gnuf64]
+# ADJUST THIS PATH TO POINT AT YOUR TOOLCHAIN
+cc = "/TOOLCHAIN_PATH/bin/loongarch64-unknown-linux-gnu-gcc"
+cxx = "/TOOLCHAIN_PATH/bin/loongarch64-unknown-linux-gnu-g++"
+ar = "/TOOLCHAIN_PATH/bin/loongarch64-unknown-linux-gnu-ar"
+ranlib = "/TOOLCHAIN_PATH/bin/loongarch64-unknown-linux-gnu-ranlib"
+linker = "/TOOLCHAIN_PATH/bin/loongarch64-unknown-linux-gnu-gcc"
+```
+
+## Cross-compilation
+
+This target can be cross-compiled on a `x86_64-unknown-linux-gnu` host. Cross-compilation on other hosts may work but is not tested.
+
+## Testing
+
+Currently there is no support to run the rustc test suite for this target.
```

Comment:
Is there an emulator/QEMU support for this architecture? That would be good to mention in this doc.

Reply:
@wesleywiser  Thanks a lot, I have updated the code.

> Is there an emulator/QEMU support for this architecture? That would be good to mention in this doc.

A simple example of testing Loongarch64 architecture binaries on the x86_86 platform: [test expample](https://github.com/rust-lang/rust/commit/8e8dee62ebbf1847e69b9442bfc9207760a00a6f)
If the rust repository needs to run related tests on the loongarch64 architecture, we can also provide remote physical machines.